tools/downloader/list_topologies.yml

```diff
@@ -1378,6 +1536,36 @@ topologies:
     framework: "tf"
     license: https://raw.githubusercontent.com/tensorflow/models/master/LICENSE
 
+  - name: "mobilenet-v2-1.0-224"
+    description: >-
+      `mobilenet-v2-1.0-224` is one of MobileNets - small, low-latency, low-power
+      models parameterized to meet the resource constraints of a variety of use cases.
+      They can be built upon for classification, detection, embeddings and segmentation
+      similar to how other popular large scale models are used. For details see paper
+      <https://arxiv.org/abs/1704.04861>
+    task_type: "classification"
+    files:
+      - name: mobilenet_v2_1.0_224.tgz
+        sha256: 318084bc1b63d6d7b854553e09cdf77078b1c0168be27c59a0d44253b5ed49dc
+        source: https://storage.googleapis.com/mobilenet_v2/checkpoints/mobilenet_v2_1.0_224.tgz
+    output: "classification/mobilenet/v2/tf/mobilenet_v2_1.0_224/"
+    postprocessing:
+      - $type: unpack_archive
+        format: gztar
+        file: mobilenet_v2_1.0_224.tgz
+    model_optimizer_args:
+      - --framework=tf
+      - --data_type=FP32
+      - --reverse_input_channels
+      - --input_shape=[1,224,224,3]
+      - --input=input
+      - --mean_values=input[127.5,127.5,127.5]
+      - --scale_values=input[127.00001206500114]
```

Comment:
Why such a strange number? In the description it says just `127.0`.

Reply:
This number was taken from benchmark output, where it is stored as 1/127.0 (and probably was rounded), and then converted to 1/(1/127.0).